datasimulator/main.py

```diff
@@ -144,9 +143,13 @@ def run_submission_order_generation(graph, data_path, node_name=None):
     else:
         submission_order = graph.generate_submission_order()
 
-    with open(os.path.join(data_path, "DataImportOrderPath.txt"), "w") as outfile:
-        for node in submission_order:
-            outfile.write(node.name + "\t" + node.category + "\n")
+    file_path = os.path.join(data_path, "DataImportOrderPath.txt")
+    path_exists = os.path.exists(data_path)
+    if not path_exists:
+        logger.error(f"Cannot create file because path does not exist. Here is the path we expect: '{data_path}'")
```

Comment:
Raise an exception with this error instead - I believe we talked about this and came to the conclusion logger.error was best but hindsight 20/20 so I think raising an exception here is better

1. Its an error state if the directory isn't there
2. Its easier to write tests that an exception was thrown vs not thrown 
3. Its consistent with how we handle other error states in data simulator

Reply:
fixed